apps/s_server.c

```diff
@@ -2191,6 +2211,45 @@ static int sv_body(int s, int stype, unsigned char *context)
         SSL_set_tlsext_debug_arg(con, bio_s_out);
     }
 
+    if (early_data) {
+        int write_header = 1, edret = SSL_READ_EARLY_ERROR;
+        size_t readbytes;
+
+        while (edret != SSL_READ_EARLY_FINISH) {
+            for (;;) {
```

Comment:
Hmm. I thought our standard was for no spaces here. I just did a regex search and found 151 instances without spaces and 4 with - so I think the no spaces have it.